### PR TITLE
Greenwithbuildingsbranch

### PR DESCRIPTION
**Pull request by jzoon**
"As an agent
I want to build green constructions
to improve my indicator score"

### CODE REVIEW

File: TU-Agent/buildConstruction.mod2g [hunk 1]
```diff
@@ -6,7 +6,8 @@ order=linear.
 module buildConstruction {
 	% Build an education construction at an available place.
 	% Repercept the relevant areas
-	if bel(functions(Functions), member(['luxe variant',BuildingID,List],Functions), relevant_areas(0,[[MultiPolygon, Area]|Tail]), randomFloor(Floors))
+	if bel(functions(Functions), member(['luxe variant',BuildingID,List],Functions), 
+		relevant_areas(0,[[MultiPolygon, Area]|Tail]), randomFloor(Floors))
```
Comment (tvleest) [inline TU-Agent/buildConstruction.mod2g]: Might prevent merge conflicts to remove this change

File: TU-Agent/buildGreen.test2g [hunk 2]
```diff
@@ -14,12 +14,14 @@ test buildGreen with
 		never done(map_sell_land(_, _, _)).
 		never done(building_plan_upgrade(_, _)).
 		
-		%If there is land available, something is build on it
-		bel(relevant_areas(0,[[MultiPolygon, Area]|Tail])) 
+		%If there no land available to build a building, trees are built
```
Comment (tvleest) [inline TU-Agent/buildGreen.test2g:17]: there is

**Comment by tvleest** [comment]
Looks good with the trees around the buildings!
There's some merge conflicts to solve, you might wanna remove the changes in the buildconstruction module, since this is just a new line.

File: TU-Agent/demolish.mod2g [hunk 3]
```diff
@@ -8,8 +8,7 @@ module demolish {
 	% Takes the first building of the list of destroyable buildings
 	% Destroys this building and deletes it of the list
 	% call get_relevant_areas to update believes about land where we can build.
-	% Also inserts the greenspace belief, to build trees on this area
```
Comment (tvleest) [inline TU-Agent/demolish.mod2g]: Removed the comment as well ?

**Comment by daanwerf** [comment]
Looks good. If it still works after the previous PR (because goal) then i approve.

**Pull request merged**